### PR TITLE
Update CSS

### PR DESCRIPTION
**Pull request by kathrynljackson**
# Iteration Number
0/1

## Link to Iteration
- [x] Did you check all of the functionality of the webpage
- [ ] Do both parties understand every line of code
- [ ] Is the code DRY/SRP

## Questions
* How do we spread the cards out separately?

# Notes
* We need to measure the width of the content box.
* Icons added and placed in the comment box.
* Still need to add hamburger icon.

## Problems
* Can't change font of text under idea box, don't know why.
* ~~The form just disappears sometimes???~~

## Screenshot 

@kathrynljackson
@taylorjohnson141
@aemcdonald

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -113,13 +118,24 @@ body {
   display: flex;
   flex: 1;
   flex-direction: column;
-}
```
Comment (taylorjohnson141) [inline styles.css:116]: Might be the cause of some of the issues, not sure though

**Review by taylorjohnson141 — CHANGES_REQUESTED** [review]
I think the only thing that seems to be off is line 116. Everything else looks great, I love the file change

File: styles.css (hunk 1 above)
Comment (aemcdonald) [reply styles.css:116]: good catch Taylor

**Review by aemcdonald — APPROVED** [review]

**Review by taylorjohnson141 — APPROVED** [review]
Looks great now!

**Pull request merged**